packages/ui/popup/content.css

```diff
@@ -37,15 +37,16 @@
 .container {
   z-index: 2;
 
-  padding: var(--gridSize) calc(2 * var(--gridSize));
+  padding: calc(2 * var(--gridSize));
+  padding-bottom: 0;
```

Comment:
@AlexBabusanu i do not agree with this padding being removed from the bottom. It might cause more problems than it fixes. confirm that it has to be removed.

Reply:
I is required to make the parity with `fe-core`, I agree that we should revisit this and make it better at a later date